### DISCUSSION TITLE
remove one component of one station

### DISCUSSION THREAD

**Discussion by vdionicio** [nikosT/Gisola#33] [General]
Dear @nikosT,

I haven't be able to remove just one component of one station when I'm revising the inversion, I had tried with some command similar to the one that you especified in the wiki  (./gisola.py --event noa2020diego -c config_dense.yaml --station ATH KLV APE LIA..NE
), but I couldn't, if I let it like this it did not exclude the component, I just can exclude the all station (all the components).

I hope you can give suggestions to solve the problem.

Thanks a lot.

Best wishes,

Viviana

**Comment by nikosT** [comment]
Dear @vdionicio,
I cannot reproduce this issue. It seems to be working fine in an example that I just tried, e.g. `python3 gisola.py -c ../test/config.yaml -e noa2020vipzs --revise --sta ATH LIA KARP KSL..NE`
which kept these 4 stations and from the last one (KSL) it kept only the N and E.

However, since the initial run (in my case) had no N in KSL (removed due to filters), it only passed the E component finally. 

Could you please be more specific in your case? Could you also attach some screenshots or scripts?
Btw, you said that you tried to revise the event, however, no `--revise` or `-r` flag is given.

Thank you in advance,
Nikos
